src/leap/mail/imap/tests/test_imap.py

```diff
@@ -435,8 +447,12 @@ def testHierarchicalRename(self):
         """
         Try to rename hierarchical mailboxes
         """
```

Comment:
yes, thanks for noticing. rename's broken, I expect to fix the testsuites while we're doing some generic QA on the internal test branch.